### DISCUSSION TITLE
LiBH4

### DISCUSSION THREAD

**Discussion by natape** [IBEXImagingCommunity/ibex_imaging_knowledge_base#55] [General]
Dear friends! 
Could you please suggest where it is possible to buy LiBH4 1g? 
Preferably Europe.
I am in Sweden, and I was informed that only 10g is available. For safety reasons I am not allowed to buy so a big amount.

**Comment by radtkea** [comment]
Thanks for your question. 

We buy it from Strem Chemicals. It looks like 1 gram aliquots ship to Sweden. I recommend that you call them to confirm. 
https://www.strem.com/catalog/v/93-0397/36/lithium_16949-15-8

Please reach out if you have any other questions. Our colleagues in the UK have not had difficulty sourcing 1 gram aliquots from Strem Chemicals.

Best of luck!
Andrea

**Comment by natape** [comment]
Thank you, Andrea! The order has been done and must come in 2 weeks.


Hämta Outlook för iOS<https://aka.ms/o0ukef>
________________________________
Från: Andrea Radtke ***@***.***>
Skickat: Friday, April 21, 2023 7:20:05 PM
Till: IBEXImagingCommunity/ibex_imaging_knowledge_base ***@***.***>
Kopia: Nataliia Petryk ***@***.***>; Author ***@***.***>
Ämne: Re: [IBEXImagingCommunity/ibex_imaging_knowledge_base] LiBH4 (Discussion #55)

You don't often get email from ***@***.*** Learn why this is important<https://aka.ms/LearnAboutSenderIdentification>

Thanks for your question.

We buy it from Strem Chemicals. It looks like 1 gram aliquots ship to Sweden. I recommend that you call them to confirm.
https://www.strem.com/catalog/v/93-0397/36/lithium_16949-15-8

Please reach out if you have any other questions. Our colleagues in the UK have not had difficulty sourcing 1 gram aliquots from Strem Chemicals.

Best of luck!
Andrea

—
Reply to this email directly, view it on GitHub<https://github.com/IBEXImagingCommunity/ibex_imaging_knowledge_base/discussions/55#discussioncomment-5688751>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/A6NMFGVDY4SLCJJW2NROAZTXCK6ULANCNFSM6AAAAAAXF4NIRE>.
You are receiving this because you authored the thread.Message ID: ***@***.***>



När du skickar e-post till Karolinska Institutet (KI) innebär detta att KI kommer att behandla dina personuppgifter. Här finns information om hur KI behandlar personuppgifter<https://ki.se/medarbetare/integritetsskyddspolicy>.


Sending email to Karolinska Institutet (KI) will result in KI processing your personal data. You can read more about KI’s processing of personal data here<https://ki.se/en/staff/data-protection-policy>.
